test/simple_test/simple_test.c

```diff
@@ -21,11 +21,10 @@
 #include <math.h>
 #include "../../dmtx.h"
 
-int
```

Comment:
but there is still many formatting changes here (while real code changes is kinda two lines as I see?)